bag_unpack:
  ros__parameters:
    input_bag_path: "/home/jakub/ros2_ws/src/bag_unpack/my_bag"
    output_bag_path: "/home/jakub/ros2_ws/src/bag_unpack/my_bag/filtered0"
    included_topics:
        - "/camera/image"
        - "/sensor/imu"
        - "/tf"
    excluded_tf:
        - "map:2:odom"
        - "base_link:2:camera_link"

